### PR TITLE
add some RegEx and JS Methods resources

### PR DESCRIPTION
**Pull request by wcjameson**
Just added a few resources, still getting comfortable with the site and the process.

### CODE REVIEW

File: content/javascript/RegEx.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
++++
+title = "RegEx"
+weight = 1
++++
+
+
+### RegularExpressions
```
Comment (wjwat) [inline content/javascript/RegEx.md:7]: Add space between Regular and Expressions

File: content/javascript/RegEx.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
++++
+title = "RegEx"
+weight = 1
```
Comment (wjwat) [inline content/javascript/RegEx.md:3]: Weights determine the order of pages in the sidebar. Having two pages with the same weight sorts them alphabetically IIRC. Is this what you intended? Also adding an `updated` field here would be nice for both of these files. Ex: `updated = 2022-03-26`

**Review by wjwat — CHANGES_REQUESTED** [review]
Hey, these look great! Aside from the comments I left already I'd like to keep file names for pages lower dash case please. So instead of `RegEx.md` use `regex.md` or `reg-ex.md` Fix those few things and we'll get this merged! Thanks for contributing 👍

File: content/javascript/RegEx.md (hunk 2 above)
Comment (wcjameson) [reply content/javascript/RegEx.md:3]: Oh cool, I did not realize.  Good to know.

**Review by wjwat — APPROVED** [review]
Looks great!

**Pull request merged**